src/main/app/common/calculate-month-increment/calculate-month-increment.ts

```diff
@@ -0,0 +1,7 @@
+import * as moment from 'moment'
+
+export function calculateMonthIncrement (d: moment.Moment) {
```

Comment:
Minor - do you think rename "d" to currDate or something is useful? I assume it is current date we have to pass!